### PR TITLE
update glbinding to v3.1.0-p0

### PR DESCRIPTION
**Pull request by tnixeu**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/tnixeu/hunter-old/builds/28399183
  * https://travis-ci.org/tnixeu/hunter_old/builds/603215173

Please fork [glbinding](https://github.com/cginternals/glbinding). The install step still needs to be changed.

### CODE REVIEW

**Comment by bkotzz** [comment]
Sure, would you like a fork of the hunter-packages repo or the upstream one?

**Comment by tnixeu** [comment]
I forked again from upstream, so I would prefer upstream.

**Comment by bkotzz** [comment]
https://github.com/cpp-pm/glbinding/tree/hunter-v3.1.0

File: cmake/projects/glbinding/hunter.cmake [hunk 1]
```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 
 # List of versions here...
 
+hunter_add_version(
+    PACKAGE_NAME
+    glbinding
+    VERSION
+    "3.1.0-p0"
+    URL
```
Comment (bkotzz) [inline cmake/projects/glbinding/hunter.cmake]: This is still the upstream release - you still need to make changes to the fork right?

Comment (rbsheth) [reply cmake/projects/glbinding/hunter.cmake]: @tnixeu *ping*

Comment (tnixeu) [reply cmake/projects/glbinding/hunter.cmake]: It should point to the patched fork of glbinding.
I have made a pull request here: https://github.com/cpp-pm/glbinding/pull/1

Comment (bkotzz) [reply cmake/projects/glbinding/hunter.cmake]: Released here https://github.com/cpp-pm/glbinding/releases/tag/hunter-v3.1.0

Comment (rbsheth) [reply cmake/projects/glbinding/hunter.cmake]: @tnixeu Please use this release instead: https://github.com/cpp-pm/glbinding/releases/tag/v3.1.0-p0

Comment (tnixeu) [reply cmake/projects/glbinding/hunter.cmake]: This PR now uses the cpp-pm fork. Thx for forking and merging.

**Pull request merged**